scripts/benchmark-rule.mjs

```diff
@@ -1,12 +1,13 @@
 'use strict';
 
 /* eslint-disable no-console */
-const Benchmark = require('benchmark');
-const got = require('got');
-const normalizeRuleSettings = require('../lib/normalizeRuleSettings');
-const postcss = require('postcss');
-const rules = require('../lib/rules');
-const { bold, yellow } = require('picocolors');
+import Benchmark from 'benchmark';
+import fetch from 'node-fetch';
+import normalizeRuleSettings from '../lib/normalizeRuleSettings.js';
+import picocolors from 'picocolors';
+import postcss from 'postcss';
+import rules from '../lib/rules';
```

Comment:
[suggestion] How about specifying the file name to remove the `--experimental-specifier-resolution=node` flag?

```suggestion
import rules from '../lib/rules/index.js';
```

Reply:
Ah smart, completely slipped my mind. Will do!